canonical/ubuntu.com
# WP-10615 get product list

## Done

- Create a mock product list and apply values in the summary section

## QA

1. Go to /pro/distributor
2. Initiate order
3. On /pro/distributor/shop, complete all steps and change values and check the change is applied in the summary section

## Issue / Card

Fixes #https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?selectedIssue=WD-10615
